transhumandesign/kag-base
# Christmas cheer

## Status

** HOLD **

## Description

Add Christmas holiday cheer!

## Steps to Test or Reproduce

Join any server during Christmas

Reason of on hold: 
1) trees created in onRestart will not have proper sectors after nextmapping; I believe Vamist has a fix we could use for this Christmas (but it changes vanilla more than I would like..)
2) all the christmas sprites are gone on nextmap (except trees, but they have another issue)
3) bushes sprite is not changing

I'm out of time so I hope someone else can take a look so we can have this for Christmas.

To-Do: Force everyone's head to be santa head

Special thanks to:
Franek for sprites
Golden for code
Vamist for code related help in #modding 

<details> 
  <summary>spoiler: changes</summary>

- Winter/Christmas textures
- Snow
- In CTF and TTH: every 5 minutes a maximum of 2 gifts appear on the map; one on each half.
- 

Preview:
![image](https://user-images.githubusercontent.com/4616537/50058871-f934b980-017e-11e9-9f70-fd2d45a489fa.png)

</details>
